cfg/pgrapher/experiment/pdsp/params.jsonnet

```diff
@@ -131,6 +131,11 @@ base {
                        // theoretical elec resp (14mV/fC): 36.6475 ADC*tick/1ke
       shaping: 2.2 * wc.us,
     },
+    // elec: super.elec {
+    //     type: "JsonElecResponse",
+    //     filename: "bnl-coldelec-response-gain14-shaping5.json.bz2",
+    //     postgain: 1.0,
+    // },
```

Comment:
If this file has a more than just a saved version of calling the `ColdElec` function then we should make its use the default and add the file to `wire-cell-data`.  If the file is just a save of the built-in function and the comment is meant just as an example usage, then it's okay as-is.

Reply:
This is just an example of usage. I've tried to test with the tabulated function to mimic ColdElecResponse (see attached figure).
![image](https://user-images.githubusercontent.com/10663117/149553660-1a36f39f-51f0-4736-a9e4-7b44080e02e4.png)
The actual usage will be implemented for the VD coldbox test once Slavic provides me with his function.